app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
I couldn't work out how to create a view in flask sqlalchemy - happy to take advice though. Ideally it would either:

- create the model, not the the table, and then the view would be created with with the query
- create the view and model all in flask sqlalchemy

This is the same for the priority view, which is much more complicated, and probably a bit of a pain to write in an ORM.